pkg/virtctl/credentials/common/common.go

```diff
@@ -57,7 +59,7 @@ func (s *SshCommandFlags) AddToCommand(cmd *cobra.Command) {
 
 func GetSshKey(flags *SshCommandFlags) (string, error) {
 	if flags.SshPubKeyLiteral != "" {
-		return flags.SshPubKeyLiteral, nil
+		return strings.TrimSpace(flags.SshPubKeyLiteral), nil
```

Comment:
What happens if multiple keys are passed here? Is it intended to support only a single key?

Reply:
Good question. Currently it would validate only the first key, then it would write all the keys into a secret.

I will change it so that multiple keys are supported.

Reply:
I've changed it so only one key is supported.

Reply:
I just wanted to make sure multiple keys are trimmed correctly. I think we should not limit the keys to just one.